projectEuler/problem5.js

```diff
@@ -0,0 +1,28 @@
+// prompt: find the smallest positive number that is evenly divisible by all of the numbers from 1 to 20
+
+// helper function for checking if integer is divisible by all numbers from 1 to 20
```

Comment:
You can shorten this comment to just `Checks if integer is divisible by all factors from 1 to 20`